src/components/DateTemperature/MainTemperatureBlock/index.js

```diff
@@ -0,0 +1,60 @@
+import "macro-css";
+import { monthNames } from "../../../utilities/constants/cities"
```

Comment:
why are your months in cities constants? the constants should names in uppercase